src/components/actividades/ListadoActividades.js

```diff
@@ -47,6 +49,36 @@ export default function ListadoActividades() {
     setAbrirModal(true);
   };
 
+  const cuposConSuColor = (anotados, cantidadMax) => {
```

Comment:
El nombre de los componentes va en mayúscula, y lo de `conSuColor` es redundante.

```suggestion
  const Cupo = (anotados, cantidadMax) => {
```

Reply:
Para convertirlo en un componente, deberías hacer que reciba props, y no dos parámetros.